plugins/com.contrastsecurity.ide.rest.sdk/.classpath

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry exported="true" kind="lib" path="lib/concurrentlinkedhashmap-lru-1.4.2.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/unbescape-1.1.3.RELEASE.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/contrast-sdk-java-3.3.2.jar" sourcepath="lib/contrast-sdk-java-3.3.2-sources.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/lombok-1.16.8.jar"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="output" path="bin"/>
+	<classpathentry exported="true" kind="lib" path="lib/lombok-1.16.8.jar"/>
+	<classpathentry kind="src" path="src/"/>
+	<classpathentry exported="true" kind="lib" path="lib/contrast-sdk-java-3.3.2.jar" sourcepath="lib/contrast-sdk-java-3.3.2-sources.jar"/>
```

Comment:
if java sdk is added through maven pom, why is this line required?

Reply:
The maven plug in updates the lib directory then the tycho plugin connector updates the config files needed in the PDE project that is the reason that this values appears in the classpath file